wcfsetup/install/files/style/icon/vendor/_variables.scss

```diff
@@ -0,0 +1,1368 @@
+// Variables
+// --------------------------
+
+$fa-font-path:                "../font/" !default;
```

Comment:
Check whether it's possible to define these variables somewhere else (so that we can just copy over the `.scss` files whenever we want to update).